aws/amazon-ecs-agent
# Log acs/tacs endpoint for connection issues

### Summary


This PR is used to log acs/tacs endpoint information which would be helpful to debug connection issues between Agent and the dependency, especially for multi tasks on same host.

### Implementation details


- Add a new field for acs endpoint in [websocket client](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/wsclient/client.go) and the [acs session](https://github.com/aws/amazon-ecs-agent/blob/master/ecs-agent/acs/session/session.go)

### Testing



New tests cover the changes: 

No

#### Manual test
```
// Launch a task and we can see when the connection is closed, we are able to see the endpoint we use
Connection closed for a valid reason: websocket: close 1000 (normal): ConnectionExpired: Reconnect to continue
```

### Description for the changelog


Enhancement - log acs/tacs endpoint

### Additional Information

**Does this PR include breaking model changes? If so, Have you added transformation functions?**
  

No

**Does this PR include the addition of new environment variables in the README?**


No
### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
